crates/xlinectl/src/command/watch.rs

```diff
@@ -1,17 +1,22 @@
+#![allow(clippy::let_underscore_untyped)]
```

Comment:
To beautify the code, otherwise I have to write something like : `let _ : &mut Command = cmd.env(key, value)`, especially this `: &mut Command` type annotation. I think it's unnecessary and not elegant, so I add this lint after checking its safety.

Reply:
What I mean is, if you put #![allow(clippy::let_underscore_untyped)] at the very beginning of the file, it means that this lint is allowed for all the code in the entire file.

Reply:
All right! I'll go and place it at right place.